velox/docs/monthly-updates/december-2023.rst

```diff
@@ -0,0 +1,99 @@
+********************
+December 2023 Update
+********************
+
+Documentation
+=============
+
+* Add documentation for :doc:`Runtime Metric</metrics>`.
+
+Core Library
+============
+
+* Add support for spilling during the output processing stage of the ``OrderBy`` operator.
+* Add support for ``k range frames`` in ``Window`` operator.
+* Add support for aggregations over sorted inputs to ``StreamingAggregation``.
+* Add support for TTL in AsyncDataCache and SsdCache. :pr:`6412`
+* Add support for TypeSignature Parser using Flex and Bison. This is used to parse function signatures.
+* Add support for metrics related to memory arbitration and spilling. :pr:`7940`, :pr:`8025`
+* Add ``Status`` class that can be used to carry the success or error state of an operation.
+  This is similar to `arrow::Status <https://arrow.apache.org/docs/cpp/api/support.html#_CPPv4N5arrow6StatusE>`_.
+* Add support for ``cast(double|real as varchar)`` to return scientific notation when magnitude
```

Comment:
This is Presto-specific, I think. Perhaps, move to Presto Functions section. CC: @gggrace14